src/lib/snack-bar/snack-bar-ref.ts

```diff
@@ -11,6 +11,12 @@ import {Observable} from 'rxjs/Observable';
 import {Subject} from 'rxjs/Subject';
 import {MatSnackBarContainer} from './snack-bar-container';
 
+/** Event that is emitted when a snack bar is dismissed. */
+export interface MatSnackBarAfterDismiss {
```

Comment:
How about just `MatSnackBarDismiss` or `MatSnackBarDismissal`? The "after" in the stream name makes sense because it relates to when the event is emitted, but there's nothing about the event object that's specific to it being at that time

Reply:
Renamed to `MatSnackBarDismiss`.